app/components/selections/ProjectStatusSelectInput/styles.module.css

```diff
@@ -0,0 +1,15 @@
+.status-input {
+    position: relative;
+}
+
+.popup {
+    display: flex;
+    flex-direction: column;
+    padding: var(--spacing-md);
+    gap: var(--spacing-lg);
+
+    .button {
+        display: flex;
+        gap: var(--spacing-lg);
+    }
+}
```

Comment:
We don't need to do manual styling, this style equivalent can be achieved by using ListLayout`  component